Libraries/PermissionsAndroid/PermissionsAndroid.js

```diff
@@ -20,7 +21,7 @@ export type Rationale = {
   buttonNeutral?: string,
 };
 
-type PermissionStatus = 'granted' | 'denied' | 'never_ask_again';
+type PermissionStatus = 'granted' | 'denied' | 'never_ask_again' | string;
```

Comment:
Why are you adding `string` ?

Reply:
We have had exact declared 3 types:
https://github.com/facebook/react-native/blob/803e993e6a8e68c62870e27ce4a45b485e244ec4/ReactAndroid/src/main/java/com/facebook/react/modules/permissions/PermissionsModule.java#L39-L41

Reply:
@retyui I've added this because of [this method](https://github.com/Krizzu/react-native/blob/spec-androidpermissions/Libraries/PermissionsAndroid/PermissionsAndroid.js#L132) resolve with `PermissionStatus`. I'm not quite sure if I could [add literals in Spec](https://github.com/Krizzu/react-native/blob/spec-androidpermissions/Libraries/PermissionsAndroid/NativePermissionsAndroid.js#L19), so I ended up with adding `string` to the type.

Reply:
@Krizzu 
Now `Spec` is source of truth

You should move `PermissionStatus` type into `Libraries/PermissionsAndroid/NativePermissionsAndroid.js`

Reply:
@retyui
Gotcha. Done that.